index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <nav id="navbar">
+      <a href="#projects">Welcome section</a>
+    </nav>
+
+    <section id="welcome-section">
+      <h1>Welcome to my portfolio page!</h1>
+    </section>
+    <div id="projects">
+      <h1 class="project-tile">Test</h1>
+      <a href="https://google.com">Visit Google</a>
+    </div>
+    <a id="profile-link" href="https://github.com/BucurEva87" target="_blank"
+      >My github profile page</a
+    >
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
```

Comment:
Please have your code pass all 12 test cases. Currently, it's only passing 11 of them.